tests/test_check_runner.py

```diff
@@ -0,0 +1,330 @@
+import sys
+import os
+import time
+from contextlib import contextmanager
+from io import StringIO
+from app import set_stage
+import stage
+import sqs_utils
+import run_result
+import check_utils
+import decorators
+from conftest import *
+
+
+def delay_rerun(*args):
+    time.sleep(20)
+    return True
+
+
+pytestmark = [pytest.mark.skip]  # don't work at the moment
+
+
+# thanks to Rob Kennedy on S.O. for this bit of code
+@contextmanager
+def captured_output():
+    """
+    Capture stdout and stderr
+    """
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
+
+class TestCheckRunner:
+    environ = DEV_ENV
+    # NOTE: this interaction is broken and needs to be refactored.
+    # The integrated dev stage runner will not poll the test stage queues! - Will Feb 23 2022
+    # set_stage('test')
+    # connection.connections['es'] = None  # disable es
+    # set up a queue for test checks
+    queue_name = stage.Stage(FOURSIGHT_PREFIX).get_queue_name()
+    runner_name = stage.Stage(FOURSIGHT_PREFIX).get_runner_name()
+    sqs = sqs_utils.SQS(FOURSIGHT_PREFIX)
+    queue = sqs.get_sqs_queue()
+
+    def clear_queue_and_runners(self, app_utils_obj):
+        """
+        Ensure that the SQS queue is empty and give existing check runners
+        a chance to finish
+        """
+        tries = 0
+        found_clear = True
+        while tries < 10:
+            sqs_attrs = self.sqs.get_sqs_attributes(self.queue.url)
+            vis_messages = int(sqs_attrs.get('ApproximateNumberOfMessages'))
+            invis_messages = int(sqs_attrs.get('ApproximateNumberOfMessagesNotVisible'))
+            if vis_messages == 0 and invis_messages == 0:
+                # allow existing checks to terminate via long polling
+                if found_clear:
+                    break
+                time.sleep(12)
+                found_clear = True
+            elif invis_messages > 0:
+                # if orphaned messages are in the queue, eat them up
+                print("%d orphaned messages at in the queue" % invis_messages)
```

Comment:
```suggestion
                print(f"{invis_messages} orphaned messages at in the queue")
```

Maybe also use `PRINT`?